config/crl_config.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+type CRLConfig struct {
+	CacheDisabled bool `json:"cacheDisabled,omitempty"`
```

Comment:
@yizha1 wanna confirm when users don't configure the CRL cache, should we enable it or disable it by default?

Reply:
As discussed, the default status of cache is `disabled`

Reply:
I would switch the variable naming back to `cacheEnabled`